WordPress/Classes/Networking/AccountServiceRemoteREST.m

```diff
@@ -80,14 +80,16 @@ - (RemoteUser *)remoteUserFromDictionary:(NSDictionary *)dictionary
 
 - (NSArray *)remoteBlogsFromJSONArray:(NSArray *)jsonBlogs
 {
-    NSMutableArray *remoteBlogs = [NSMutableArray arrayWithCapacity:[jsonBlogs count]];
-    for (NSDictionary *jsonBlog in jsonBlogs) {
-        BOOL isJetpack = [jsonBlog[@"jetpack"] boolValue];
-        if (!isJetpack || JetpackREST.enabled) {
-            [remoteBlogs addObject:[self remoteBlogFromJSONDictionary:jsonBlog]];
-        }
+    NSArray *blogs = jsonBlogs;
+    if (!JetpackREST.enabled) {
```

Comment:
If `JetpackREST.enabled`, the previous filter would always add the mapped object to the array, so it was only filtering if that was false. Unless I'm missing something, the behavior should still be the same.